photon-targeting/src/main/native/include/photon/targeting/PhotonPipelineResult.h

```diff
@@ -120,6 +120,7 @@ class PhotonPipelineResult {
 
   /**
    * Returns a reference to the vector of targets.
+   * <p> Returned in the order set by target sort mode. </p>
    * @return A reference to the vector of targets.
```

Comment:
Nit: this isn't really "a reference", that's not a thing in Java. Should probably be re-worded?

Reply:
The Java docs say its a copy which is probably more accurate

Reply:
Ok sorry this is what I get for reviewing code on my phone. This is actually a reference, it's just a span. We should consider changing this to be a span of const targets, but that's out of scope. Can you put it back to the reference verbiage? And leave Java as copy?

Although Java is weird. It's a list of the same underlying target objects, so I suspect that modifying the ones in the returned list mutate the result too? whack